src/Grids/latitude_longitude_grid.jl

```diff
@@ -252,6 +252,8 @@ Adapt.adapt_structure(to, grid::LatitudeLongitudeGrid{FT, TX, TY, TZ}) where {FT
 ####### Utilities to precompute Metrics 
 #######
 
+metrics_precomputed(grid::LatitudeLongitudeGrid{FT, TX, TY, TZ, M}) where {FT, TX, TY, TZ, M} = !(M<:Nothing) 
```

Comment:
This might be slightly easier to understand --- your call

```suggestion
metrics_precomputed(::LatitudeLongitudeGrid{<:Any, <:Any, <:Any, <:Any, Nothing}) = false 
metrics_precomputed(::LatitudeLongitudeGrid) = true
```

Reply:
Do we also need a type like

```julia
const PrecomputedMetricsLatLonGrid = LatitudeLongitudeGrid{<:Any, <:Any, <:Any, <:Any, Nothing}
```

?

Reply:
I don't think we need it because that information is not required outside of spacing_areas_and_volumes so maybe a type for that would be a little redundant